WARNING  2025-03-03 09:45:23 [pytest_result_log:122]  : --------------Start: testcases/test_api.py::TestApi::test_get_tag---------------
WARNING  2025-03-03 09:45:23 [pytest_result_log:128]  : ---------------End: testcases/test_api.py::TestApi::test_get_tag----------------